cabal-testsuite/PackageTests/CustomSegfault/cabal.test.hs

```diff
@@ -1,6 +1,5 @@
 import Test.Cabal.Prelude
 main = cabalTest $ do
-    -- TODO: this test ought to work on Windows too
-    skipUnless "not Linux" =<< isLinux
+    skipUnless "depends on `unix` and needs Linux" isLinux
```

Comment:
For the reviewer: It certainly doesn't build on Windows, I don't know if it works on Mac though